src/Interactive/HostTest/InteractiveHostTests.cs

```diff
@@ -986,8 +987,8 @@ public void Script_NoHostNamespaces()
         {
             Execute("nameof(Microsoft.CodeAnalysis)");
 
-            AssertEx.AssertEqualToleratingWhitespaceDifferences(@"
-(1,8): error CS0234: The type or namespace name 'CodeAnalysis' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?)",
+            AssertEx.AssertEqualToleratingWhitespaceDifferences($@"
+(1,8): error CS0234: { string.Format(CSharpResources.ERR_DottedTypeNameNotFoundInNS, "CodeAnalysis", "Microsoft") }",
```

Comment:
>  [](start = 22, length = 1)

We do not add spaces around expressions in interpolated strings anywhere in the Roslyn repo. It would be good to keep the style consistent.